src/rules/alias.ts

```diff
@@ -49,8 +49,7 @@ function findAlias(
 		// Remove basedir and slash in start
 		const slicedImportPath = importPath
 			.slice(baseDir.length + 1)
-			.slice(path.dirname(matchedPath).length + 1);
-
+			.slice(path.dirname(path.normalize(matchedPath)).length + 1);
```

Comment:
Why this change for? What problem (maybe potential case?) we have with current code?

Reply:
Without normalizing, if you have a tsconfig path like:
`"@e2e/*": ["./cypress/e2e/*"],`

The slicing logic will return the string missing two characters, due to the additional two characters from `./`.

So the message I received was something like: `Update import to @e2e/st` instead of `Update import to @e2e/test`